include/grpcpp/impl/codegen/client_context.h

```diff
@@ -200,6 +200,13 @@ class ClientContext {
   /// end in "-bin".
   /// \param meta_value The metadata value. If its value is binary, the key name
   /// must end in "-bin".
+  ///
+  /// Metadata must conform to the following format:
+  /// Custom-Metadata → Binary-Header / ASCII-Header
```

Comment:
Not a fan of using unicode in C code. I don't think we've done it this far, so can we keep that a `->` rather than a `→` ?

Reply:
Done. Thanks for catching that - it was an unintentional copy paste error :)